fastlane/Fastfile

```diff
@@ -108,7 +108,7 @@ platform :ios do
   desc 'Creates a new release branch from the current develop'
   lane :code_freeze do |options|
     gutenberg_dep_check
-    old_version = ios_codefreeze_prechecks(options)
+    ios_codefreeze_prechecks(options)
```

Comment:
`old_version` was a leftover of when we generated the PRs list locally. See d2dd89ec9f